specs/eip4844/polynomial-commitments.md

```diff
@@ -117,9 +141,51 @@ def bit_reversal_permutation(sequence: Sequence[T]) -> Sequence[T]:
 ```python
 def bytes_to_bls_field(b: Bytes32) -> BLSFieldElement:
     """
-    Convert bytes to a BLS field scalar. The output is not uniform over the BLS field.
+    Convert 32-byte value to a BLS field scalar. The output is not uniform over the BLS field.
+    """
+    return int.from_bytes(b, ENDIANNESS) % BLS_MODULUS
+```
+
+#### `blob_to_field_elements`
+
+```python
+def blob_to_field_elements(blob: Blob) -> Polynomial:
```

Comment:
Optional: can change to `blob_to_polynomial`

Reply:
Done in 2e5a784d6